demo/src/main/res/layout/merge_player_controls.xml

```diff
@@ -3,15 +3,18 @@
   xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
   android:layout_height="wrap_content"
-  tools:parentTag="LinearLayout">
+  tools:parentTag="android.widget.LinearLayout">
 
   <ImageView
     android:id="@+id/player_controls_play_pause"
-    android:layout_width="48dp"
-    android:layout_height="48dp"
-    android:layout_marginLeft="12dp"
-    android:layout_marginRight="12dp"
-    android:src="@drawable/play" />
+    android:layout_width="@dimen/play_button_size"
+    android:layout_height="@dimen/play_button_size"
+    android:layout_marginLeft="@dimen/player_controls_padding_horizontal"
+    android:layout_marginStart="@dimen/player_controls_padding_horizontal"
```

Comment:
match this to the ones below by calling it `margin_start`

Reply:
Well this is effectively a padding of the controls view group rather than a margin of a single item, hence the name.